Core/src/org/sleuthkit/autopsy/commonfilesearch/CommonFilesPanel.java

```diff
@@ -164,6 +166,9 @@ protected void done() {
         "CommonFilesPanel.search.results.titleAll=Common Files (All Data Sources)",
         "CommonFilesPanel.search.results.titleSingle=Common Files (Match Within Data Source: %s)",
         "CommonFilesPanel.search.results.pathText=Common Files Search Results",
+        "CommonFilesPanel.search.done.searchProgress1=Gathering Common Files Search Results.",
+        "CommonFilesPanel.search.done.searchProgress2=Generating Common Files Search Results.",
+        "CommonFilesPanel.search.done.searchProgress3=Displaying Common Files Search Results.",
```

Comment:
These should me more descriptively named than just searchProgress1, 2, 3.

Reply:
Updated variable names.